klusterletconfig/v1alpha1/types.go

```diff
@@ -0,0 +1,82 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=klusterletconfigs
```

Comment:
is this cluster scoped?

Reply:
`// +kubebuilder:resource:scope=Cluster` added.